sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -177,6 +177,109 @@ def begin_recognize_receipts_from_url(self, receipt_url, **kwargs):
+        if content_type == "application/json":
+            raise TypeError("Call begin_recognize_business_cards_from_url() to analyze a business card from a URL.")
+
+        include_field_elements = kwargs.pop("include_field_elements", False)
+
+        if content_type is None:
+            content_type = get_content_type(business_card)
+
+        try:
+            return self._client.begin_analyze_business_card_async(  # type: ignore
+                file_stream=business_card,
+                content_type=content_type,
+                include_text_details=include_field_elements,
+                cls=kwargs.pop("cls", self._receipt_callback),
+                polling=True,
+                **kwargs
+            )
+        except ValueError as e:
+            if "begin_analyze_business_card_async" in str(e):
+                raise ValueError(
+                    "Method 'begin_recognize_business_cards' is only available for API version V2_1_PREVIEW and up"
+                )
+            raise e
+
+    @distributed_trace
+    def begin_recognize_business_cards_from_url(
+            self,
+            business_card_url,
+            **kwargs
+    ):
+        # type: (str, Any) -> LROPoller[List[RecognizedForm]]
+        """Extract field text and semantic values from a given business card.
+        The input document must be the location (URL) of the card to be analyzed.
+
+        See fields found on a business card here:
+        https://aka.ms/formrecognizer/businesscardfields
+
+        :param str business_card_url: The URL of the business card to analyze. The input must be a valid, encoded URL
+            of one of the supported formats: JPEG, PNG, PDF and TIFF.
+        :keyword str locale: Locale of the business card. Default locale is en-US. Other supported locales
+            include: en-AU, en-CA, en-GB, and en-IN.
+        :keyword bool include_field_elements:
+            Whether or not to include field elements such as lines and words in addition to form fields.
+        :keyword int polling_interval: Waiting time between two polls for LRO operations
+            if no Retry-After header is present. Defaults to 5 seconds.
+        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :return: An instance of an LROPoller. Call `result()` on the poller
+            object to return a list[:class:`~azure.ai.formrecognizer.RecognizedForm`].
+        :rtype: ~azure.core.polling.LROPoller[list[~azure.ai.formrecognizer.RecognizedForm]]
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+        include_field_elements = kwargs.pop("include_field_elements", False)
+
+        try:
+            return self._client.begin_analyze_business_card_async(  # type: ignore
+                file_stream={"source": business_card_url},
+                include_text_details=include_field_elements,
+                cls=kwargs.pop("cls", self._receipt_callback),
```

Comment:
same here

Reply:
and other places just in case

Reply:
yeah we can definitely rename this to something more general like `_prebuilt_callback` since both receipt/business cards use it.

Reply:
good point @maririos , I got too lazy initially, thanks!